eox_hooks/tests/tutor/integration_test_tutor.py

```diff
@@ -1,14 +1,21 @@
 """
 Test integration file.
 """
-from django.test import TestCase
+from django.test import TestCase, override_settings
 
 
+@override_settings(ALLOWED_HOSTS=['local.edly.io', 'testserver'], SITE_ID=2)
```

Comment:
`local.edly.io` is it still necessary since the `testserver` was included?

Why is `SITE_ID=2` necessary, could you explain?

Reply:
Because I was getting the following error: django.contrib.sites.models.Site.DoesNotExist: Site matching query does not exist.

When an instance is running, the SITE_ID is 2, so the settings were overridden
so that during the tests it also takes the SITE_ID = 2